src/table/interfaces.tsx

```diff
@@ -466,6 +466,7 @@ export namespace TableProps {
   export interface LiveAnnouncement {
     firstIndex: number;
     lastIndex: number;
+    visibleItemsCount: number;
```

Comment:
This is a change to the public API, and this new prop is not optional, so not a backwards-compatible change? This is used by `renderAliaLive`.

Reply:
Thanks, that is indeed an API change! I will remove it from the current PR, it is meant to be released separately: https://github.com/cloudscape-design/components/pull/2165

Reply:
Are you still aiming to remove this from this PR? Or rather preferring to merge #2165 first now?

Reply:
I will merge 2165 first, it is blocked by the dry run atm.

Reply:
@jperals updated